### PR TITLE
OSDOCS-4561: Removed Latency Sensitivity references from vSphere UPI …

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-4561](https://issues.redhat.com/browse/OSDOCS-4561)

Version(s):
4.14 to 4.10

Link to docs preview:
* [Installing RHCOS and starting the OpenShift Container Platform bootstrap process](https://61358--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-machines_installing-vsphere)
* [Addng more compute machines to a cluster in vSphere](https://61358--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#machine-vsphere-machines_installing-vsphere)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-machines.adoc [hunk 1]
```diff
@@ -165,9 +165,13 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
+[comment]
+--
+//Need to replace with alternative text to give a possible use case of diagnosing cluster performance issues. Possible a resource pool configuration.
```
Comment (dfitzmau) [inline modules/installation-vsphere-machines.adoc:170]: Query for SME: VMC docs exist for 4.10 to 4.13, so I should remove any Latency Sensitivity references from these docs also?

File: modules/machine-vsphere-machines.adoc [hunk 2]
```diff
@@ -42,7 +42,6 @@ Ensure that all virtual machine names across a vSphere installation are unique.
 .. Optional: On the *Select storage* tab, customize the storage options.
```
Comment (Timdawson264) [inline modules/machine-vsphere-machines.adoc:42]: Can we also remove this `Optional:` this has been a point of contention.

Comment (dfitzmau) [reply modules/machine-vsphere-machines.adoc]: Thanks, @Timdawson264 . I removed this sentence from both `machine-vsphere-machines.adoc` and `installation-vsphere-machines.adoc` files.

File: modules/installation-vsphere-machines.adoc (hunk 1 above)
Comment (jcpowermac) [reply modules/installation-vsphere-machines.adoc:169]: Yeah I would think so

File: modules/installation-vsphere-machines.adoc [hunk 3]
```diff
@@ -165,10 +164,14 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
...
 *** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
 **** Memory reservation value must be equal to its configured memory size.
-**** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
-*** Click *Edit Configuration*, and on the *Configuration Parameters* window, search the list of available parameters for steal clock accounting (`stealclock.enable`). If it is available, set its value to `TRUE`. Enabling steal clock accounting can help with troubleshooting cluster issues.
+--
+*** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
```
Comment (jcpowermac) [inline modules/installation-vsphere-machines.adoc:173]: 1. Latency sensitivity should be completely removed, we should not be even optionally recommending this setting.
2. IMHO we should not be suggesting to a user to set reservations directly on a guest, this should be in a resource pool. Setting at the guest level should be a last resort.
3. I do not believe we should be telling users what the allocation for memory and cpu reservations should be. This has to be configured by the customers' vSphere administrator based on the physical design and the guests running in that cluster. Those prescribed settings could have serious impacts to the vSphere enviornment though not as severe as latency sensitivity to high. Anything we state should be suggestions only and not *must*.

**Comment by dfitzmau** [comment]
I identified a possible [misleading link](https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-machines_installing-vsphere). Maybe this link is genuine, but the user should be formed that when they click on the link, a zip file downloads to their local system?

Screenshot from 2023-07-12 14-40-00

File: modules/installation-vsphere-machines.adoc (hunk 3 above)
Comment (dfitzmau) [reply modules/installation-vsphere-machines.adoc]: Thanks, @jcpowermac . @jcpowermac  and @Timdawson264 . I removed some content and marked section 9.f as optional. I also adding a disclaimer that places the onus on a cluster admin to ensure they configure their cluster accordingly. Please let me know your thoughts. 

I'm not sure how granular I need to be with the resource pool suggestions or if the resource pool could be a focal point for removed steps?

File: modules/installation-vsphere-machines.adoc [hunk 4]
```diff
@@ -157,26 +165,18 @@ $ export IPCFG="ip=<ip>::<gateway>:<netmask>:<hostname>:<iface>:none nameserver=
...
+*** Click *Edit Configuration*, and on the *Configuration Parameters* window, search the list of available parameters for steal clock accounting (`stealclock.enable`). Set the parameter to the value of `TRUE`. Enabling steal clock accounting can help with troubleshooting cluster issues.
 *** Click *Add Configuration Params*. Define the following parameter names and values:
-**** `guestinfo.ignition.config.data`: Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type.
-**** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
 **** `disk.EnableUUID`: Specify `TRUE`.
 **** `stealclock.enable`: If this parameter was not defined, add it and specify `TRUE`.
+**** Create a child resource pool from the cluter's root resource pool. Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type.
```
Comment (jcpowermac) [inline modules/installation-vsphere-machines.adoc:173]: ignition has no association with resource pools

ignition is set per virtual machine regardless of location in vsphere

File: modules/installation-vsphere-machines.adoc [hunk 5]
```diff
@@ -157,26 +165,18 @@ $ export IPCFG="ip=<ip>::<gateway>:<netmask>:<hostname>:<iface>:none nameserver=
...
-... Set the `guestinfo.afterburn.initrd.network-kargs` property before booting a VM from an OVA in vSphere:
-+
-[source,terminal]
-----
-$ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
-----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
-**** Memory reservation value must be equal to its configured memory size.
-**** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
```
Comment (Timdawson264) [inline modules/installation-vsphere-machines.adoc:170]: **** CPU reservation value must be at least the number of virtual CPUs multiplied by the measured physical CPU speed.

Comment (Timdawson264) [reply modules/installation-vsphere-machines.adoc:170]: Sorry i dont understand why CPU reservation has been removed, yes you can remove the refrence to low latency Please add back the above...

**Review by Timdawson264 — CHANGES_REQUESTED** [review]

File: modules/installation-vsphere-machines.adoc (hunk 5 above)
Comment (dfitzmau) [reply modules/installation-vsphere-machines.adoc:170]: Hi @Timdawson264 . This PR needs to move forward, so I've removed the CPU reservation statement. Please file a separate docs bug Jira if you want the statement added back, where we can also work together on a warning statement for the customer to proceed with this configuration.

**Comment by nalhadef** [comment]
LGTM, Darragh!

**Pull request merged**